### PR TITLE
Feat/url chain specific

### LINKED ISSUE #165: Separate domains with network-specific URL routing
Make different domains accessible through network-specific routing.

Example:
For gemini-2c: `https://subspace.github.io/blockexplorer/gemini-2c/blocks`
For gemini-3d:  `https://subspace.github.io/blockexplorer/gemini-3d/blocks`

```[tasklist]
### Tasks
- [ ] https://github.com/subspace/blockexplorer/issues/173
- [ ] https://github.com/subspace/blockexplorer/issues/174
- [ ] https://github.com/subspace/blockexplorer/issues/175
- [ ] https://github.com/subspace/blockexplorer/issues/177
```

```[tasklist]
### Acceptance criteria
- [x] The URL specifies the network
- [ ] Legacy links fallback to `gemini-2c`
- [x] Search placeholder prompt the network
- [ ] Make the networks name properly identify the version, sample: `Gemini 3c`
- [x] UI remembers which network was previously selected
- [x] UI remembers the selected network if the tab or window was closed
```

### LINKED ISSUE #173: Change the routing to match the selected network

### LINKED ISSUE #174: Make naming conventions for network display

### LINKED ISSUE #175: Default to Gemini 3c on page load

### LINKED ISSUE #177: UI Should Remember which network has the user selected

### PR DESCRIPTION
**Pull request by natachadelarosa**
#165 
close #173 
close #174 
close #175 
close #177

### CODE REVIEW

**Comment by isSerge** [comment]
if you visit an internal page and then refresh or share a link - it will lead to the home page, instead of an internal page, for example, https://deploy-preview-176--precious-crostata-1376e8.netlify.app/#/gemini-3b/extrinsics/0000322351-000010-92e88

File: client/src/layout/config/chains.json [hunk 1]
```diff
@@ -1,14 +1,16 @@
 [
   {
-    "title": "Gemini II",
+    "title": "Gemini III",
     "urls": {
-      "api": "https://squid.gemini-2a.subspace.network/graphql"
+      "api": "https://squid.gemini-3b.subspace.network/graphql",
+      "page": "gemini-3b"
```
Comment (isSerge) [inline client/src/layout/config/chains.json:6]: It's actually `gemini-3c`, I was re-using the same URL, but moving forward I will probably create a new droplet and a new URL for each iteration

**Comment by natachadelarosa** [comment]
> if you visit an internal page and then refresh or share a link - it will lead to the home page, instead of an internal page, for example, https://deploy-preview-176--precious-crostata-1376e8.netlify.app/#/gemini-3b/extrinsics/0000322351-000010-92e88

Yeah for this, I'll use your suggestion of saving the user network selection

**Comment by isSerge** [comment]
> > if you visit an internal page and then refresh or share a link - it will lead to the home page, instead of an internal page, for example, https://deploy-preview-176--precious-crostata-1376e8.netlify.app/#/gemini-3b/extrinsics/0000322351-000010-92e88
> 
> Yeah for this, I'll use your suggestion of saving the user network selection

It's different, I mean `extrinsics/0000322351-000010-92e88` part is excluded when user opens the page

**Comment by natachadelarosa** [comment]
> > > if you visit an internal page and then refresh or share a link - it will lead to the home page, instead of an internal page, for example, https://deploy-preview-176--precious-crostata-1376e8.netlify.app/#/gemini-3b/extrinsics/0000322351-000010-92e88
> > 
> > 
> > Yeah for this, I'll use your suggestion of saving the user network selection
> 
> It's different, I mean `extrinsics/0000322351-000010-92e88` part is excluded when user opens the page

Yeah, I know, it is part of it because I added a redirect for this cases of page reload or that is affecting link opening as well

File: client/src/App.tsx [hunk 2]
```diff
@@ -40,75 +40,73 @@ function ScrollToTopWrapper({ children }) {
 
 // force to refresh url on browser refresh
 // TODO: this is a temporary solution, we should use a better solution
-function UpdateURLOnRefresh({ children }) {
-  const location = useLocation()
-  const navigate = useNavigate()
-  const { selectedChain } = useDomains()
+// function UpdateURLOnRefresh({ children }) {
```
Comment (isSerge) [inline client/src/App.tsx:43]: do we still need this?

File: client/src/layout/config/chains.json (hunk 1 above)
Comment (isSerge) [reply client/src/layout/config/chains.json:6]: could you update this one to `gemini-3c`?

**Pull request merged**